scrapper_config.json

```diff
@@ -1,4 +1,4 @@
 {
-    "seed_urls": [],
+    "seed_urls": ["http://www.vestnik-mslu.ru/vestnik.asp?vest_lang=Rus&vest_type=gum"],
```

Comment:
were we talking about changing the resource?

Reply:
yes, it's an old version, right now I'm working on the new resourse